cli/kill.go

```diff
@@ -9,16 +10,23 @@ import (
 
 func init() {
 	register("kill", runKill, `
-usage: flynn kill <job>
+usage: flynn kill ID...
 
-Kill a job.`)
+Kill running jobs.`)
 }
 
 func runKill(args *docopt.Args, client controller.Client) error {
-	job := args.String["<job>"]
-	if err := client.DeleteJob(mustApp(), job); err != nil {
-		return err
+	success := true
+	for _, id := range args.All["ID"].([]string) {
+		if err := client.DeleteJob(mustApp(), id); err != nil {
+			success = false
+			log.Printf("could not kill job %s: %s\n", id, err)
```

Comment:
Prefix this log message with `ERROR: ` so it is more obvious to the user that the job did not get killed.